lib/node_modules/@stdlib/array/docs/types/index.d.ts

```diff
@@ -489,7 +489,7 @@ interface Namespace {
 	empty: typeof empty;
 
 	/**
-	* Creates an uninitialized array having the same length and data type as a provided input array.
+	* Creates an uninitialized array having the same length as a provided input array.
```

Comment:
@Planeshifter Looks like we lost part of the description in https://github.com/stdlib-js/stdlib/commit/819d2e407146d4dcc17f8bab53b591b3d573f8a1, as, by default, we do infer the data type from the input array.